packages/docs/src/web/Chip/Chip.mdx

```diff
@@ -0,0 +1,36 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import { Chip } from '@naturacosmeticos/natds-web';
+
+<Meta title="Web|Chip" component={Chip} />
+
+# Chip
+
+Chips are compact elements that represent an input, attribute, or action.
+
+### Usage
+
+Chips allow users to enter information, make selections, filter content, or trigger actions. Chips should appear dynamically as a group of multiple interactive elements. Unlike buttons, which should be a consistent and familiar call to action, one that a user expects to appear as the same action in the same general area.
+
+***
+
+### Principles
+
+***
+
+#### Compact
+
+Chips are compact components that represent discrete information.
+
+#### Relevant
+
+Chips should have a clear and helpful relationship to the content or task they represent.
+
+#### Focused
+
+Chips should make tasks easier to complete, or content easier to sort.
+
+<Preview>
+  <Chip label="label" />
+</Preview>
+
+<Story id="web-chip--default" />
```

Comment:
Change to extension `.story.mdx`

Reply:
You can use it both as a story or as a Doc.